comit/src/rsa.rs

```diff
@@ -0,0 +1,115 @@
+//! Recommended Safe Actions
+//!
+//! This module provides logic for recommending when it is safe/unsafe to take a
+//! particular action for the COMIT protocol.
+//!
+//! 'Safe' is defined as:
+//! 1 - You will not loose money.
+//! 2 - The counterparty is likely to continue with the swap because it is safe
+//!     for them to do so.
+
+use crate::Timestamp;
+use time::Duration;
+
+/// SwapTime enables recommendations on whether it is safe to take an action.
+#[derive(Debug, Clone, Copy)]
+pub struct SwapTime {
```

Comment:
What is the reasoning behind the name? I don't find it particularly descriptive for the functionality it exposes :see_no_evil:

Reply:
Before this I had `Foo` :) I'm always open to suggestions for better names.